composer.json

```diff
@@ -55,5 +52,11 @@
     },
     "conflict": {
         "zendframework/zend-barcode": "*"
-    }
+    },
+    "repositories": [
+        {
+            "type": "git",
+            "url": "https://github.com/pelex/ZendPdf.git"
```

Comment:
We'll need to get rid of this before merging

Reply:
It looks like I'm able to run all on PHP 8.1 now. How to resolve this. Can I add this legacy package to packagist?

Reply:
Good question: is it not in laminas yet? 🤔

Reply:
Missing in laminas project or I'm not able to find it.

Reply:
The original component (zendframework/zendpdf) was abandoned on 2017-03-01 and was therefore not ported to Laminas. And the unit tests for the PDF renderer have also been deactivated:

https://github.com/laminas/laminas-barcode/blob/cdca9bf632df2174fd5aa624dcc273f1c3fe3932/phpunit.xml.dist#L21

Reply:
I could try rewriting this pdf renderer to tecnickcom/tcpdf.